src/registry/stats.rs

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: Apache-2.0
+
+//! Compute stats on a semantic convention registry.
+
+use clap::Args;
+use weaver_cache::Cache;
+use weaver_logger::Logger;
+use weaver_resolver::attribute::AttributeCatalog;
+use weaver_resolver::registry::resolve_semconv_registry;
+use weaver_resolver::SchemaResolver;
+
+/// Parameters for the `registry stats` sub-command
+#[derive(Debug, Args)]
+pub struct StatsRegistry {
+    /// Local path or Git URL of the semantic convention registry.
+    pub registry: String,
+
+    /// Optional path in the Git repository where the semantic convention
+    /// registry is located
```

Comment:
For my understanding, this is generally the `model` directory?

Reply:
I left this part configurable because in the future we could allow the creation of vendor or enterprise oriented registries.

Reply:
#32 created